fetchApp.js

```diff
@@ -0,0 +1,15 @@
+async function getData() {
+  try {
+    const response = await fetch(`https://catfact.ninja/fact`);
+    const data = await response.json();
+    console.log(data);
+    for (const obj of data) {
+      if (obj) {
+        console.log(obj);
+      }
+    }
+  } catch (error) {
+    console.log(error);
+  }
+}
+getData();
```

Comment:
Good start! I like how you use `async/await` instead of `fetch/then`! You got the data, you just had to insert elements in the DOM using that data, and to call this function three times! I see that again, it is the creation of elements in JS that is the thing that stopped you. Google always helps in these situations, and in this exam it is encouraged to use it. So make sure to look up how to do this!